### ISSUE TITLE
Monkey's Audio file format not supported

### ISSUE THREAD

**Issue by gitivn93** [alexkay/spek#303]
I have some songs in .ape format and spek can't read them to analyze their spectrum. I would like to see support for this format in a future update.

Thanks in advance

**Comment by itsTyrion** [comment]
can confirm.
FLAC:
![flac](https://github.com/alexkay/spek/assets/17730120/df909ccd-d3af-4f3b-9453-9e5f4ed4f15e)

Monkey's Audio (compression set to normal):
![ape-normal](https://github.com/alexkay/spek/assets/17730120/e59dce49-6774-4a6c-8094-7b785f0fe5e6)

the really quiet passages seem to be correct, but most it...... not. 

Interestingly, it gets worse on the "Very High" compression level:
![ape-veryhigh](https://github.com/alexkay/spek/assets/17730120/c91046f4-f6ff-421f-bb28-068f80985b4e)

**Comment by gitivn93** [comment]
Indeed, spek seems to get stuck or give problems with songs in .ape format. Possibly it is due to the type of compression used on those songs. Although some songs seem to read correctly, this behavior is very strange.

**Comment by kedaitinh12** [comment]
Try this version
https://github.com/MikeWang000000/spek-X
